adobe_vipm/flows/validation/transfer.py

```diff
@@ -282,4 +329,12 @@ def validate_transfer(mpt_client, adobe_client, order):
         subscription["offerId"] = correct_sku or subscription["offerId"]
     customer = adobe_client.get_customer(authorization_id, transfer.customer_id)
     commitment = get_3yc_commitment(customer)
-    return add_lines_to_order(mpt_client, order, subscriptions, commitment, "currentQuantity")
+
+    one_time_skus = get_one_time_skus(mpt_client, order)
```

Comment:
Talked privately - there is an issue that now all one-time items are removed from transfer. It should be only in the case if all subscriptios are expird